backend/utils.py

```
@@ -0,0 +1,8 @@
+from models import Setup
```

Ruff (0.11.9)

`models.Setup` imported but unused

Remove unused import: `models.Setup`

(F401)